src/shared/utils/state.ts

```diff
@@ -0,0 +1,26 @@
+import { writeFile, readFile } from "fs/promises";
+
+// Get state/mode Harmony is in (default or mottagnings-mode)
+export async function getState(): Promise<string> {
+    let mode = "default";
+    let data;
+    try {
+        data = await readFile("./src/shared/assets/mode.txt");
```

Comment:
Would be nice if this were in a database rather than just a file. And I think this is already using one?

Reply:
> And I think this is already using one?

Already using what? A database? As far as I know, I don't think we are.

#### Context
The file `mode.txt` contains information on whether mottagningsmode is on or off (a boolean). 

Is it worth having a database for storing just a single boolean? Perhaps. Might we expand with more database variables in the future? Perhaps. Are there any other things we might want to put in there? I don't know; what do you guys think? 

Tagging relevant people for the discussion: @nylanderdev @Benjaneb @Carlltz @Hampfh @niklasvatn 

**PS:** If you think a DB is the right solution and you know how to set it up, help would be appreciated!

Reply:
Oh - I read [this](https://github.com/datasektionen/harmony/blob/46c9d4a7e375d3c892d04b440e7f7f389f0a3023/src/database-config.ts#L4C1-L6C34) earlier and thought it was backed by a database, since the library supports it, but it seems it's just in memory which wouldn't work very well here.

But yeah then it is not worth setting it up for just one boolean :p

Reply:
If we're saving this on file I suggest that we at least make it a .json file since that will make it directly compatible to use in code which will simplify things and make it a little more neatly organized

Reply:
> If we're saving this on file I suggest that we at least make it a .json file since that will make it directly compatible to use in code which will simplify things and make it a little more neatly organized

Implemented this now